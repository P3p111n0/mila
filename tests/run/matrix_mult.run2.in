54
53
40
60
88
98
34
65
73
31
68
21
67
91
57
100
97
41