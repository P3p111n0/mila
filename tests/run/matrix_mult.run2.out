9225 12375 5795
17556 21594 10294
12709 15308 7412
